app/src/main/java/de/danoeh/antennapod/activity/AudioplayerActivity.java

```diff
@@ -638,20 +618,12 @@ public boolean isDrawerOpen() {
 
     @Override
     public boolean onCreateOptionsMenu(Menu menu) {
```

Comment:
If we're not doing anything here, is there any need to override this function (or the next)?